packages/i18nify-js/scripts/generate-root-imports.js

```diff
@@ -0,0 +1,35 @@
+/**
+ * This script generates sub-module files at the root level.
+ * For eg, For module currency, it will create i18nify-js/currency.js with below snippet
+ * export * from './lib/esm/currency';
+ *
+ * If we already have exports in package.json for sub-modules then why do we need it ?
+ * While exports specified in package.json work well for web-based projects, the React Native bundler fails to recognize them
+ * and requires additional configuration to enable recognition. Refer this issue => https://github.com/razorpay/blade/issues/2036
+ * As a result, we are generating additional files for sub-module imports to function correctly in React Native platform.
+ *
+ */
+const fs = require('fs');
+const path = require('path');
+
+// Currently, we will have to manuually add every new module that we create
```

Comment:
1. manuually --> manually
2. If this is TODO to automate this, add a TODO in beginning of comment.